HOWTOSTART.md

```diff
@@ -0,0 +1,334 @@
+# Setting up Development Environment for Origam
+The following is a guide to help you set up Origam development environment. You will need it if you wish to contribute to Origam backend or frontend. It will be quite helpful if you are going to develop your own plugin for Origam too.
+
+Origam can be developed on Windows only because the Architect application is written using Windows Forms. 
```

Comment:
I think we are using Windows on Parallels to develop the Architect on Mac. Moreover for most of the tasks we do not need to compile Architect so the development shall be possible on Mac.

I do not know if there is any free alternative for Visual Studio for Mac, whether and how we can develop and compile the server on Mac.

The frontend can be developed on Mac in VS Code with no problem.

I would also mention somewhere that you can download Architect and Server already compiled and where it is in case you do not want or need to compile them yourself.